jquantsapi/client.py

```diff
@@ -2,12 +2,14 @@
 from datetime import datetime
 from typing import Dict
 
+from typing import Union
 import pandas as pd  # type: ignore
 import requests
 from dateutil import tz
 from requests.adapters import HTTPAdapter
 from urllib3.util import Retry
 
+DatetimeLike = Union[datetime, pd.Timestamp, str]
```

Comment:
class の前には空白行を2行いれていただけるとありがたいです！

https://peps.python.org/pep-0008/#blank-lines
> Surround top-level function and class definitions with two blank lines.

```suggestion
DatetimeLike = Union[datetime, pd.Timestamp, str]

```

Reply:
レビューありがとうございます、 対応します！ 🙏